custom-fonts.md

```diff
@@ -1,13 +1,19 @@
 ---
 layout: page
-title: Using Custom Fonts
+title: Using Fonts
 
 permalink: /custom-fonts/
 ---
 
-You can use custom fonts in your Flutter application by including them in your
-`pubspec.yaml` file under the `fonts` heading in your `flutter` section:
+## Using fonts
```

Comment:
Add a sentence under this, with a description of what the user will learn, and what motivates this page.

For example: "You can use different fonts in your Flutter app. For example, you may have a custom font that your designer created, or you may have a font from Google Fonts (link)."